eng/Packages.Data.props

```diff
@@ -31,6 +31,7 @@
     <PackageReference Update="Microsoft.Azure.KeyVault.Core" Version="3.0.3" />
     <PackageReference Update="Microsoft.Azure.Management.Batch" Version="4.2.0" />
     <PackageReference Update="Microsoft.Azure.Management.ContainerRegistry" Version="2.0.0" />
+    <PackageReference Update="Microsoft.Azure.Management.CosmosDB" Version="1.1.0" />
```

Comment:
I'm not familiar with this props file. Could you explain why cosmosdb was added here?

Reply:
Talked with Meha offline. This file was for maintaining dependencies in the repo. CosmosDB's manamagement plane SDK isn't for now used in mulptiple projects, so it's not necessary.